anitya/lib/utilities.py

```diff
@@ -103,14 +103,16 @@ def check_project_release(project, session, test=False):
 
     p_versions = project.get_sorted_version_objects()
     old_version = project.latest_version or ''
+    version_column_len = models.ProjectVersion.version.property.columns[0].type.length
     for version in versions:
         if version not in p_versions:
-            project.versions_obj.append(
-                models.ProjectVersion(
-                    project_id=project.id,
-                    version=version.version
+            if len(version.version) < version_column_len:
+                project.versions_obj.append(
+                    models.ProjectVersion(
+                        project_id=project.id,
+                        version=version.version
```

Comment:
The current column size is 50 char any reason for not making this bigger ? maybe 100 ?

Reply:
It's enough for most projects, I only saw one project where this limit is not enough and this looks like mistake not a correct version. See https://github.com/jd/daiquiri/tags

Reply:
If this limit will be an issue in future we can update it, but right now it is enough.

Reply:
ok :+1: